src/Poseidon/Package.hs

```diff
@@ -262,7 +278,8 @@ readPoseidonPackageCollection opts baseDirs = do
     let loadedPackages = rights eitherPackages
     -- package duplication check
     -- This will throw if packages come with same versions and titles (see filterDuplicates)
-    finalPackageList <- liftIO $ filterDuplicatePackages loadedPackages
+    filteredPackageList <- liftIO $ filterDuplicatePackages (_readOptKeepMultipleVersions opts) loadedPackages
+    let finalPackageList = sort filteredPackageList
```

Comment:
That is my change -- as discussed

Reply:
Wait... what was this about? Ah, it's resolved... OK